openshift/tests-extension/test/qe/specs/olmv1_cc.go

```diff
@@ -34,4 +43,1061 @@ var _ = g.Describe("[sig-olmv1][Jira:OLM] clustercatalog", g.Label("NonHyperShif
 
 	})
 
+	g.It("PolarionID:69123-[Skipped:Disconnected]Catalogd clustercatalog offer the operator content through http server", g.Label("LEVEL0"), func() {
```

Comment:
Why do we need this "PolarionID:69123"
Those tests are not sending a signal to the Sippy 
Also, I understand that they are not using the old system anymore

So, could we remove it and keep the names meaningful?

Reply:
Why we are adding `g.Label("LEVEL0")` ?